platform-configs/include/default.robot

```diff
@@ -14,7 +14,7 @@ ${RTE_S2_N_PORT}=                                   13541
 ${FLASH_SIZE}=                                      ${TBD}
 ${FLASH_LENGTH}=                                    ${TBD}
 ${TIANOCORE_STRING}=                                to boot directly
-${BOOT_MENU_KEY}=                                   F11
+${BOOT_MENU_KEY}=                                   ${F11}
```

Comment:
Same kind of change would have to be done in `novacustom-common.robot`, might as well change it now as it is the last place that needs to be fixed.

Reply:
added in: a04c99c94b4d73b9b8034667e5be29ce077ee958